src/main/resources/assets/opencomputers/loot/OpenOS/bin/hostname.lua

```diff
@@ -0,0 +1,14 @@
+local args = {...}
+if args[1] then
+  local file = io.open("/etc/hostname", "w")
+  file:write(args[1])
```

Comment:
Better a `file, reason = ...` with a `if not file then io.stderr:write(reason .. "\n") else ...` here to avoid comparatively cryptic nil errors on read-only file systems / when out of disk space?